scripts/uberenv/spack_configs/blueos_3_ppc64le_ib_p9/packages.yaml

```diff
@@ -105,6 +108,20 @@ packages:
    paths:
       tar: /usr
    buildable: False
+  spectrum-mpi:
+    paths:
+      spectrum-mpi: /usr/tce/packages/spectrum-mpi/spectrum-mpi-rolling-release-clang-coral-2018.08.08/
+    buildable: False
+  mvapich2:
+    paths:
+      mvapich2%clang: /usr/tce/packages/mvapich2/mvapich2-2020.01.09-clang-ibm-2019.10.03/
+      mvapich2%xl@coral: /usr/tce/packages/mvapich2/mvapich2-2020.01.09-xl-2020.06.25
+    buildable: False
+  netlib-lapack:
+    paths:
+      # This is the one that has the trailing underscores, needed for MFEM compat
+      netlib-lapack: /usr/tcetmp/packages/lapack/lapack-3.9.0-gcc-7.3.1/lib/
+    buildable: False
```

Comment:
You updated Spack, so now it probably warn you that this syntax is deprecated (use of `paths`).
Also, if Spack is new enough, you can now mark `mpi` as `buildable: false` which adds an extra level stability, because as-is Spack could still decide to install a fancy MPI provider in certain conditions.

Reply:
That deprecated warning is probably being lost in the very long dev-build output.  Do spack specific warnings/output go to a file like the package level spack-build.out files?